RecoLocalTracker/SiStripClusterizer/plugins/StripDataView.cuh

```diff
@@ -0,0 +1,27 @@
+#ifndef RecoLocalTracker_SiStripClusterizer_plugins_StripDataView_h
+#define RecoLocalTracker_SiStripClusterizer_plugins_StripDataView_h
+
+#include "CUDADataFormats/SiStripCluster/interface/SiStripClustersCUDA.h"
+
+#include <cstdint>
+
+struct ChannelLocsView;
+
+namespace stripgpu {
+  static constexpr auto kMaxSeedStrips = 200000;
```

Comment:
```suggestion
  HOST_DEVICE_CONSTANT auto kMaxSeedStrips = 200000;
```
should make it possible to use `kMaxSeedStrips` directly in CPU and GPU code, without the need for `__device__ constexpr int maxseeds() { return kMaxSeedStrips; }`